system/netdata.service.in

```diff
@@ -37,12 +43,28 @@ OOMScoreAdjust=1000
 # To give netdata the max priority, set CPUSchedulingPolicy=rr and CPUSchedulingPriority=99
 CPUSchedulingPolicy=idle
 
-# This sets the scheduling priority (for policies: rr and fifo).
-# Priority gets values 1 (lowest) to 99 (highest).
-#CPUSchedulingPriority=1
+NoNewPrivileges=false
+PermissionsStartOnly=true
+# CAP_SETGID is required for setgroups()
+# CAP_NET_RAW is needed by fping
+CapabilityBoundingSet=CAP_DAC_READ_SEARCH CAP_SYS_PTRACE CAP_SETGID CAP_NET_RAW
+PrivateTmp=true
+ProtectHome=read-only
+ProtectSystem=full
+
+ReadOnlyDirectories=/
```

Comment:
If netdata sees all mount points as read-only it will not report disk space usage (size, inodes).
This setting actually kills `diskspace.plugin`.

I am not sure what is right to do in this case. Leave the directories read-write or remove the detection of read-only mount points and report disk space usage even on read-only ones.